__tests__/app.test.js

```diff
@@ -79,7 +79,65 @@ describe('2. GET api articles', () => {
                 .expect(404)
                 .then ((res) => {
                     const {msg} = res.body;
-                    expect(msg).toBe('Path not found')
+                    expect(msg).toBe('Not Found')
+                })
+        })
+    })
+})
+
+describe('3. Get api/articles/:article_id', () => {
+    describe('a. status 200 & data', () => {
+        it('returns 200 status code & the article data object with the corresponding article_id back to the user', () => {
+            return request(app)
+            .get('/api/articles/3')
+            .expect(200)
+            .then( (res) => {
+                const {article} = res.body;
+                expect(article).toBeInstanceOf(Object);
+                expect(article).toEqual(
+                    expect.objectContaining({
+                        article_id : 3,
+                        title : 'Eight pug gifs that remind me of mitch',
+                        topic : 'mitch',
+                        author : 'icellusedkars',
+                        body : 'some gifs',
+                        created_at : '2020-11-03T09:12:00.000Z',
+                        votes : 0,
+                    })
+                )
+
+            })
+        })
+    })
+
+    describe('b. error handling', () => {
+        it('send a 400 status code when a user inputs a bad request', () => {
+            return request(app)
+                .get('/api/articles/article3')
+                .expect(400)
+                .then ((res) => {
+                    const {msg} = res.body;
+                    expect(msg).toBe('Bad Request')
+                })
+        })
+
+        it('send a 404 status code when user inputs a bad path', () => {
+            return request(app)
+                .get('/api/artiales/3')
+                .expect(404)
+                .then ((res) => {
+                    const {msg} = res.body;
+                    expect(msg).toBe('Not Found')
+                })
+        })
+
+        it('send a 404 status code when user inputs a valid id but data does not exist', () => {
+            return request(app)
+                .get('/api/artiales/99999')

```

Comment:
This path needs to be totally correct except the ID - make sure 'articles' is spelt right